operator/roles/forkliftcontroller/templates/populator/deployment-populator-controller.yml.j2

```diff
@@ -26,8 +26,8 @@ spec:
                   fieldPath: metadata.namespace
           - name: OVIRT_POPULATOR_IMAGE
             value: {{ populator_ovirt_image_fqin }}
-          args:
```

Comment:
@liranr23 FYI
had to remove it, because we need different ports for each controller